### DISCUSSION TITLE
How to configure "cache-control" header when request remote imgs?

### DISCUSSION THREAD

**Discussion by KyleRicardo** [vuejs/vitepress#1766] [Q&A]
Hi, I'm using Backblaze + Cloudflare to host my images. When I'm using vitepress I found the images' response header `cf-cache-status` is always `MISS`.

I checked the request header and I found `cache-control` and `pragma` values are both `no-cache`. When I imported the same request into Postman and modified both `cache-control` and `pragma` to `max-age=86400`, the response header `cf-cache-status` turned to `HIT`. I think the problem is here.

Is there any method that I can configure the cache related request header of imgs?

Many thanks!

**Comment by brc-dd** [comment]
How are you serving your site? You need to configure the headers on your hosting provider.\
(VitePress just generates the files not serve them.)

**Comment by KyleRicardo** [comment]
Thanks for the quick reply!

For now it's not deployed, I'm using `pnpm docs:dev` and debug on localhost.

However, I plan to deploy on Netlify when I finished dev.

Any suggestions?

**Comment by brc-dd** [comment]
Try deploying it directly. I don't think additional config will be required. You can configure the response headers (https://docs.netlify.com/routing/headers/); the request won't have any headers regarding cache control in prod.

**Comment by tassoman** [comment]
On GitLab pages, nowadays it's a desire.
https://gitlab.com/gitlab-org/gitlab-pages/-/issues/50#note_989858443
